Report/Chapters/Design.tex

```diff
@@ -73,7 +73,7 @@ \subsection{Behaviour and interaction design}
 
 \section{Languages and libraries}
 
-Viskell is written in Java using Java 8 with JavaFX. \index{JavaFX}
+Viskell is written in Java 8 that comes bundled with JavaFX. \index{JavaFX}
 Java 8 is chosen because it, unlike Java 7, still receives public updates at the moment of writing.
 Language level 8 is chosen to make use of some of the new features in this version.
```

Comment:
I dont know, I find the last sentence to be very unecessary, id say either combine the 2 or drop the second one